generic/tests/netperf.py

```diff
@@ -95,28 +95,38 @@ def mtu_set(mtu):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
-
-    if len(params.get("nics", "").split()) > 1:
-        session = vm.wait_for_login(nic_index=1, timeout=login_timeout)
-    else:
-        session = vm.wait_for_login(timeout=login_timeout)
-
     config_cmds = params.get("config_cmds")
     if config_cmds:
         for config_cmd in config_cmds.split(","):
             cmd = params.get(config_cmd.strip())
+            session = vm.wait_for_serial_login(timeout=login_timeout)
             if cmd:
                 s, o = session.cmd_status_output(cmd)
-                if s:
+                logging.info(o)
+                if "querysettings" in cmd:
+                    if ".sys" in o:
+                        verifier_clear_cmd = "verifier /reset"
+                        status, output = session.cmd_status_output(verifier_clear_cmd)
+                        logging.info(output)
+                        if ".sys" in output:
+                            msg = "% does not work correctly" % verifier_clear_cmd
+                            test.error(msg)
+                if s != 0:
```

Comment:
I see cmd is in a comd list, so there are more than one windows cmds,but some windows cmd's status is not accurate, such as 'verifier /reset' will return 2 or 0 which is correct.
So, I suggest delete it.

Reply:
the line 114 is for other cmd but not for "verifier /reset", we need to keep the other cmd's return is 0 .

Reply:
yeah,I know that, but it is included in 'for config_cmd in config_cmds.split(","):' loop, so my question is do you want to add other windows cmd to config_cmds list later? If not,it's okay.

If yes,it's not suitable for all windows cmds like I said previously.

Actually, if config_cmds  has only one value, there is no need to use a cmd list.

Reply:
@xiagao I add parameter config_cmds_expect_status and expect_status to ignore the exit issue. 

for which the exit is not 0 and do not rasie the error , we can set like : 
 config_cmds_expect_status = yes
 xxx_expect_status = no

Reply:
It's ok for me.
Thanks.